client/src/components/DatasetProjectCard.js

```diff
@@ -55,7 +55,14 @@ export const DatasetProjectCard = ({ dataset, projectId }) => {
       <Box>
         <Box margin={{ bottom: '24px' }}>
           <Label label="Diagnosis" />
-          <Text>{projectStats.diagnoses_counts}</Text>
+          {Object.entries(projectStats.diagnoses_counts)
+            .sort(([a], [b]) => a.localeCompare(b))
+            .map(([k, v], i) => (
+              <Fragment key={k}>
+                {i > 0 && ', '}
+                {k} ({v})
+              </Fragment>
+            ))}
```

Comment:
Can we convert this to a helper `listCounts` that takes a dict of counts and returns a string that matches how we present it on the portal. We can reuse this helper in `ProjectSearchResult` as well.

Reply:
Updated: I've added two helpers, `formatCounts` (for formatting a counts object) and `sortArrayString` (for sorting an array of strings), and used them in both components. Thank you!